### ISSUE TITLE
Model output

### ISSUE THREAD

**Issue by zaher88abd** [kelkelcheng/GC-Net-Tensorflow#3]
Hi @kelkelcheng, 
Please, I have one last question.
Why you have a 'disparity_filter', I couldn't understand the propose of it. In the paper, they just use Softmax at the end.
```   
    logits = tf.nn.softmax(neg)
    disparity_filter = tf.reshape(tf.range(0, d, 1, dtype=tf.float32), [1, 1, d, 1])
    distrib = conv2d(logits, disparity_filter, 1)
```

**Comment by zaher88abd** [comment]
Thanks, I got the idea.

**Comment by kelkelcheng** [comment]
Great, let me know if you have other questions:)

On Aug 14, 2019, at 2:52 PM, Mohamad Zaher Abd Ulmoula <notifications@github.com<mailto:notifications@github.com>> wrote:


Thanks, I got the idea.
